warehouse/macros/parsed_file_common_element_lookups.sql

```diff
@@ -0,0 +1,43 @@
+-------------- ALL --------------
+
+{% macro read_file_config_and_partitions() %}
+
+        dt,
+        hour,
+        {{ get_string_value_at_json_path('file' ,'config.url') }} AS _url,
+        {{ get_string_value_at_json_path('file' , 'config.name') }} AS _name,
+        TIMESTAMP({{ get_string_value_at_json_path('file', 'ts') }}) AS _extract_ts
+
+{% endmacro %}
+
+{% macro metadata_columns() %}
+    dt,
+    hour,
+    _url,
+    _name,
+    _extract_ts
```

Comment:
nit: probably want a trailing comma for these

Reply:
disagree, see the usage -- I think it's more natural to use a comma after the macro call if one is needed (two macros back to back with no comma look weird to me?)